### DISCUSSION TITLE
Welcome to journey-book Discussions!

### DISCUSSION THREAD

**Discussion by Adarsh-jaiss** [collab-community/journey-book#8] [Ideas]
hey, @kendalldoescoding I have invited you as a collaborator, kindly check that out. And another thing is if you do any changes,kindly let me know here , that will keep me updated regarding everything :)

**Comment by KendallDoesCoding** [comment]
> hey, @KendallDoesCoding I have invited you as a collaborator, kindly check that out. And another thing is if you do any changes,kindly let me know here , that will keep me updated regarding everything :)

Thanks... I'll do all work in another branch and make a PR, but it will be much easier to do work coz you gave me collab now!

**Comment by Adarsh-jaiss** [comment]
-- I have a question from you, is folks are going to face any problem in coding javascript because they don't know what others are coding in   HTML and CSS?

**Reply by KendallDoesCoding** [reply]
Not really, I mean, it depends... Like, in my project, a contributor helped me make a darkmode toggle button, but all the code, was only in JS for that button, he may've added one line of code in HTML to display the button, but I don't remember. Although, it is suggested the person knows HTML if they know JS atleast, CSS doesn't really apply coz that's just for styling purposes.

**Reply by Adarsh-jaiss** [reply]
Oh okay, sounds awesome 🤝
